### ISSUE TITLE
Feature Request: Key Bindings

### ISSUE THREAD

**Issue by stormasm** [nushell/reedline#245]
Right arrow is full hint completion #244

It would be great if the end user could have the ability...

**_to make TAB the full hint completion instead of Right arrow..._**

Personally for me on my macbook pro keyboard the tab is a much
easier keystroke to complete then the right arrow which is far way
in the right side of my keyboard where as TAB is right there and quick...

And everyone has a different preference so with that in mind...

This is an example of something that ideally would be configurable.

**Comment by fdncred** [comment]
i think this is already configurable. it's up to the host of reedline to configure it i believe. so e-q in our case.

**Comment by stormasm** [comment]
@fdncred ok cool...
so should this be an issue be in engine-q instead so that I can set my TAB
to accomplish the full hint completion ?

**Comment by fdncred** [comment]
yes, it should be an issue in e-q. there may already be one there - not sure.
